src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-collapsible-panel.ts

```diff
@@ -24,7 +25,22 @@ export async function addCollapsiblePanel(
 
   if (!appMenu) return;
 
-  appMenu.insertAdjacentElement('afterend', collapsiblePanelView.element);
+  const panelNodes =
+    injectionContainer.querySelectorAll<HTMLElement>('.aqn') ?? [];
```

Comment:
Would we want to add a static constant or use `CollapsiblePanelView.elementSelectors.NATIVE` here?

Reply:
Nice